physical/raft/fsm_test.go

```diff
@@ -125,3 +128,30 @@ func TestFSM_Batching(t *testing.T) {
 		t.Fatal("config wasn't updated")
 	}
 }
+
+func TestFSM_List(t *testing.T) {
+	fsm, dir := getFSM(t)
+	defer os.RemoveAll(dir)
+
+	ctx := context.Background()
+	count := 100
+	keys := rand.Perm(count)
+	var sorted []string
+	for _, k := range keys {
+		err := fsm.Put(ctx, &physical.Entry{Key: fmt.Sprintf("foo/%d/bar", k)})
```

Comment:
Could you write two values to this storage prefix so below we can make sure only one is listed?

Reply:
Done.